### ISSUE TITLE
Doors Bug on 1.16

### ISSUE THREAD

**Issue by candrawardana** [ProtocolSupport/ProtocolSupport#1340]
Server version 17
build 631

**Comment by candrawardana** [comment]
![image](https://user-images.githubusercontent.com/38995778/123782440-8c7a0880-d8ff-11eb-8ff7-7d61ad6c01f1.png)

**Comment by Shevchik** [comment]
Does this happen with all doors, or some specific door state, or only at some specific location?

**Comment by roopekoo** [comment]
I also have this issue.
My server is at version 1.17 paper.
This happens when the client is on version 1.16.2-1.16.5.
The blocks seem to bug when player interaction happens or similar: 
-piston moves a block
-Player grows a tree with bone meal
-Activating any rail with redstone so the rail block state changes
-Waterlogging a block
- Any door, (right-clicking does not restore the door because it updates)
- etc.
Basically when the blockstate changes, the block changes to be something else.

NOTE1: Right-clicking restores the incorrect block to normal
NOTE2: Other players that have the same version see the same visual bug. Players with the correct version, eg 1.17 see everything normally.
NOTE3: Every bugged block restores to original on relogging

**Comment by Shevchik** [comment]
Found why it happens. Multiple block change translation is missing the usage of flattening id table. Will fix it soon.

**Comment by candrawardana** [comment]
nice

**Comment by Shevchik** [comment]
Should be fixed by https://github.com/ProtocolSupport/ProtocolSupport/commit/ab6fe35f8bf752ee84d6b281b88add812048e496

**Comment by roopekoo** [comment]
All above mentioned issues seem to be fixed on the latest release. Thank you Shevchik!
